config/locales/ja.yml

```diff
@@ -1,15 +1,28 @@
 ---
 ja:
   about:
-    about_mastodon: Mastodon は<em>自由でオープンソース</em>なソーシャルネットワークです。商用プラットフォームの代替となる<em>分散型</em>を採用し、あなたのやりとりが一つの会社によって独占されるのを防ぎます。信頼できるインスタンスを選択してください &mdash; どのインスタンスを選んでも、誰とでもやりとりすることができます。 だれでも自分の Mastodon インスタンスを作ることができ、シームレスに<em>ソーシャルネットワーク</em>に参加できます。
+    about_mastodon_html: Mastodon は、オープンなウェブプロトコルを採用した、自由でオープンソースなソーシャルネットワークです。電子メールのように分散型の仕組みを採っています。
     about_this: このインスタンスについて
     business_email: 'ビジネスメールアドレス:'
-    closed_registrations: 現在このインスタンスでの新規登録は受け付けていません。
+    closed_registrations: 現在このインスタンスでの新規登録は受け付けていません。しかし、他のインスタンスにアカウントを作成しても全く同じネットワークに参加することができます。
     contact: 連絡先
     description_headline: "%{domain} とは？"
     domain_count_after: 個のインスタンス
     domain_count_before: 接続中
+    features:
+      humane_approach_body: 他のSNSたちの失敗から学び、Mastodon はソーシャルメディアが誤った使い方をされることの無いよう、倫理的な設計を目指しています。
+      humane_approach_title: より人道的な設計
+      not_a_product_body: Mastodon は営利的なSNSではありません。広告も、データ収集も無く、また塀に囲まれた庭でもありません。
```

Comment:
"Walled Garden" is a software term for closed off, corporate environments that don't permit any innovation or creativity. An example of this is the Apple iOS app store, where all apps are curated and approved by Apple. This is contrasted to the web, where anyone can start up a website and they don't need the approval of some large corporation like Google or Twitter.

This sentence is specifically saying that Mastodon isn't a "walled garden" because there's no central authority that can determine who sets up mastodon servers—anyone at all can buy a computer and join the network immediately, without getting anybody's permission.

does that help?